.github/workflows/linting.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   linting:
     runs-on: ubuntu-latest
     steps:
-      - uses: UCL-MIRSG/.github/actions/linting@v0.36.0
+      - uses: UCL-MIRSG/.github/actions/linting@v0.38.0
```

Comment:
One day Renovate will come back 😆

Reply:
we can only dream

Reply:
Dreams do come true - I just reenabled it...

I put it on the shared calendar to reenable, but I still forgot...

Reply:
I was scared to re-enable it haha